onnxruntime/core/providers/openvino/openvino_execution_provider.cc

```diff
@@ -201,7 +201,7 @@ bool IsOpSupported(std::string name){
 
 //Checks if the entire graph is supported by OpenVINO EP and returns false if it is not.
 
-bool IsGraphSupported(const onnxruntime::GraphViewer& graph_viewer, std::string dev_id){
+bool IsGraphSupported(const onnxruntime::GraphViewer& graph_viewer, std::string dev_id, std::string &error_msg){
```

Comment:
Use exceptions to avoid having to return a bool.
void IsGraphSupported(const onnxruntime::GraphViewer& graph_viewer, std::string dev_id) {
     if(condition) {
            throw "Error message";
     }
}